hist/hist/src/THStack.cxx

```diff
@@ -1045,3 +1045,11 @@ void THStack::SetMinimum(Double_t minimum)
    fMinimum = minimum;
    if (fHistogram) fHistogram->SetMinimum(minimum);
 }
+
+
+////////////////////////////////////////////////////////////////////////////////
+/// Get iterator over itnernal hists list.
+TIter THStack::begin() const
```

Comment:
Yes, because `TList` is not completely defined yet, so this just threw a bunch of errors for me.